specification/sections/03-evaluation-context.md

```diff
@@ -88,26 +88,28 @@ See [setting a provider](./01-flag-evaluation.md#setting-a-provider) for details
 
 ##### Conditional Requirement 3.2.2.4
 
-> The API **MUST** have a a mechanism to manage `evaluation context` for an associated name.
+> The API **MUST** have a mechanism to manage `evaluation context` for an associated name.
 
 In the static-context paradigm, it must be possible to create and remove provider-specific context.
 See [setting a provider](./01-flag-evaluation.md#setting-a-provider) for details.
 
 #### Requirement 3.2.3
 
-> Evaluation context **MUST** be merged in the order: API (global; lowest precedence) -> client -> invocation -> before hooks (highest precedence), with duplicate values being overwritten.
+> Evaluation context **MUST** be merged in the order: API (global; lowest precedence) -> transaction -> client -> invocation -> before hooks (highest precedence), with duplicate values being overwritten.
```

Comment:
Transaction context is only available when using the dynamic context paradigm but it's nice to have this be a single requirement. Perhaps we could explicitly clarify this in the non-normative section?

Reply:
I think it's fine to leave it but I would recommend adding a short explanation. FYI @toddbaert

Reply:
I added a note to this.